conf/plants/species_tree.topology.nw

```diff
@@ -79,68 +79,90 @@
+	       triticum_spelta,
                triticum_turgidum,
                triticum_aestivum_arinalrfor,
                triticum_aestivum_jagger,
                triticum_aestivum_julius,
+               triticum_aestivum_kariega,
                triticum_aestivum_lancer,
                triticum_aestivum_landmark,
                triticum_aestivum_mace,
                triticum_aestivum_mattis,
                triticum_aestivum_norin61,
+               triticum_aestivum_paragon,
+               triticum_aestivum_renan,
                triticum_aestivum_stanley,
               ),
               (
                triticum_urartu,
                (
                 triticum_aestivum_A,
                 triticum_dicoccoides_A,
+		triticum_timopheevii_A,
+                triticum_spelta_A,
                 triticum_turgidum_A,
                 triticum_aestivum_arinalrfor_A,
                 triticum_aestivum_jagger_A,
                 triticum_aestivum_julius_A,
+                triticum_aestivum_kariega_A,
                 triticum_aestivum_lancer_A,
                 triticum_aestivum_landmark_A,
                 triticum_aestivum_mace_A,
                 triticum_aestivum_mattis_A,
                 triticum_aestivum_norin61_A,
+                triticum_aestivum_paragon_A,
+                triticum_aestivum_renan_A,
                 triticum_aestivum_stanley_A,
                ),
               ),
               (
                triticum_aestivum_B,
-               triticum_dicoccoides_B,
+	       triticum_dicoccoides_B,
+	       triticum_timopheevii_B,
+               triticum_spelta_B,
                triticum_turgidum_B,
                triticum_aestivum_arinalrfor_B,
                triticum_aestivum_jagger_B,
                triticum_aestivum_julius_B,
+               triticum_aestivum_kariega_B,
                triticum_aestivum_lancer_B,
                triticum_aestivum_landmark_B,
                triticum_aestivum_mace_B,
                triticum_aestivum_mattis_B,
                triticum_aestivum_norin61_B,
+               triticum_aestivum_paragon_B,
+               triticum_aestivum_renan_B,
                triticum_aestivum_stanley_B,
               ),
               (
                aegilops_tauschii,
+               triticum_spelta_D,
                triticum_aestivum_D,
                triticum_aestivum_arinalrfor_D,
                triticum_aestivum_jagger_D,
                triticum_aestivum_julius_D,
+               triticum_aestivum_kariega_D,
                triticum_aestivum_lancer_D,
                triticum_aestivum_landmark_D,
                triticum_aestivum_mace_D,
                triticum_aestivum_mattis_D,
                triticum_aestivum_norin61_D,
+               triticum_aestivum_paragon_D,
+               triticum_aestivum_renan_D,
                triticum_aestivum_stanley_D,
               ),
               (
                triticum_aestivum_U,
+               triticum_spelta_U,
```

Comment:
Would it be possible to remove this? The removal of `triticum_spelta_U` would make the species topology more closely match the Compara `genome_db` data for `triticum_spelta`.
```suggestion
```

Reply:
I thought this has been committed. May I added it?